Xamarin.Forms.Build.Tasks/SetPropertiesVisitor.cs

```diff
@@ -992,16 +1000,97 @@ static IEnumerable<Instruction> ConnectEvent(VariableDefinition parent, string l
 			yield return Create(Callvirt, module.ImportReference(adder));
 		}
 
+		static bool CanConnectCommandToEvent(VariableDefinition parent, string localName, INode valueNode, bool attached, ILContext context)
+		{
+			var module = context.Body.Method.Module;
+
+			if (attached)
+				return false;
+			if (parent.VariableType.GetEvent(ed => ed.Name == localName, out _) == null)
+				return false;
+			if (!context.Variables.TryGetValue(valueNode as IElementNode, out VariableDefinition varValue))
+				return false;
+			//TODO: check if parent is a BP
+			if (varValue.VariableType.InheritsFromOrImplements(module.ImportReference(("Xamarin.Forms.Core", "Xamarin.Forms", "BindingBase"))))
+				return true;
+			//if (varValue.VariableType.InheritsFromOrImplements(module.ImportReference(("netstandard", "System.Windows.Input", "ICommand"))))
```

Comment:
Should this still be here? please delete, uncomment, or explain why commented out. I think it should be uncommented.